api/v1/store_env.go

```diff
@@ -350,16 +350,15 @@ func (s *Store) GetEnv() []corev1.EnvVar {
 				},
 			},
 		},
-		// When using https we need this for header trusting
-		{
-			Name:  "TRUSTED_PROXIES",
-			Value: "REMOTE_ADDR",
-		},
 		// Changed in 6.7
 		{
 			Name:  "SYMFONY_TRUSTED_PROXIES",
 			Value: "REMOTE_ADDR",
 		},
+		{
+			Name:  "SYMFONY_TRUSTED_HEADERS",
+			Value: "['x-forwarded-for', 'x-forwarded-host', 'x-forwarded-proto', 'x-forwarded-port']",
```

Comment:
we should test this before (I could imagine that they are just comma seperated) and allow also allow overwriting it by the user